modules/cluster-logging-loki-grafana.adoc

```diff
@@ -0,0 +1,89 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: PROCEDURE
+[id="logging-loki-grafana_{context}"]
+
+= Configuring Grafana
+Grafana is an established web-based dashboard and visualization tool for interacting with Loki.
+
+.Prerequisites
+
+* {logging-title-uc}: 5.5
+* `LokiOperator`
+* Logs forwarding to LokiStack
+
+
+.Using the Gateway With OpenShift-based Authentication
+The preferred option for accessing the data stored in Loki managed by loki-operator when running on OpenShift with the default OpenShift tenancy model is to go through the LokiStack gateway and do proper authentication against the authentication service included in OpenShift.
+
+An example configuration authenticating to the gateway in this manner is available in addon_grafana_gateway_ocp_oauth.yaml.
```

Comment:
This file currently exists in the `loki` repository. How will this be available in the documentation? Linked, as a download or inline?